File: Wrap-Up/python/lowest_common_multiple.py
def lowest_common_multipe_calc(first_number, second_number):

    start = first_number
    if (second_number > first_number):
        start = second_number
      

        for count in range(start, first_number * second_number):
            if (count % first_number == 0 and count % second_number == 0):
                return count
          
    return first_number * second_number




first_number = 10
second_number = 20
print(lowest_common_multipe_calc(first_number, second_number))

File: Wrap-Up/javaScript/lowestCommonMultiple.js
function lowestCommonMultipeCalc(firstNumber, secondNumber){



 
let start = firstNumber;
    if (secondNumber > firstNumber) {
        start = secondNumber;
        }

        for (let count = start; count <= firstNumber * secondNumber; count++) {
            if (count % firstNumber == 0 && count % secondNumber == 0) {
                return count; 
            }
        }

       return firstNumber * secondNumber;


}

firstNumber = 10;
secondNumber = 20;
console.log(lowestCommonMultipeCalc(firstNumber, secondNumber));
